pjsip/include/pjsua2/account.hpp

```diff
@@ -706,9 +706,27 @@ struct AccountNatConfig : public PersistentObject
      * Default constructor
      */
     AccountNatConfig() : sipStunUse(PJSUA_STUN_USE_DEFAULT),
-			 mediaStunUse(PJSUA_STUN_USE_DEFAULT),
-			 nat64Opt(PJSUA_NAT64_DISABLED),
-			 turnConnType(PJ_TURN_TP_UDP)
+      mediaStunUse(PJSUA_STUN_USE_DEFAULT),
+      nat64Opt(PJSUA_NAT64_DISABLED),
+      iceEnabled(false),
+      iceTrickle(PJ_ICE_SESS_TRICKLE_DISABLED),
+      iceMaxHostCands(-1),
+      iceAggressiveNomination(true),
+      iceNominatedCheckDelayMsec(PJ_ICE_NOMINATED_CHECK_DELAY),
+      iceWaitNominationTimeoutMsec(ICE_CONTROLLED_AGENT_WAIT_NOMINATION_TIMEOUT),
+      iceNoRtcp(false),
+      iceAlwaysUpdate(true),
+      turnConnType(PJ_TURN_TP_UDP),
+      contactRewriteUse(true),
```

Comment:
As the type is `int`, should be better to initialized with `1` (or perhaps `PJ_TRUE`), this is similar to few fields below.

Optionally, while in this area, also update the docs of all `int` members in this class if their default value are not integer, e.g: changing `TRUE` to `1 (yes)`. Thanks.

Reply:
`iceNoRtcp` and `iceAlwaysUpdate` are `bool`s 
Good point re `contactRewriteUse` I've updated to `PJ_TRUE`

Have updated the docs to the int fields.